server/models/Payment.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+const Sitter = required('./Sitter.js');
+
+const Payment = new mongoose({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'User',
+  },
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'Sitter',
+  },
+  rate: {
+    type: Number,
+    required: true,
+    set: getRate,
+  },
+  hoursOfService: {
+    type: Number,
+    required: true,
+  },
+  totalPayment: {
+    type: Number,
+    required: true,
+    set: payTotalOf,
+  },
+  customerId: {
+    type: String,
+    required: true,
+  },
+});
+
+const getRate = async (sitterId) => {
+  const sitter = await Sitter.findById(sitterId);
+  return sitter.rate;
+};
```

Comment:
I don't thing this is really needed, we could just grab the rate when fetching a pet sitter.

Reply:
i changed that its a getter now